File: Latihan B/Java/Disk.java
public class Disk{

	//atribut private
	private String type; //tipe disk
	private int capacity; //kapasitas disk
	private int price; //harga disk


	//konstruktor
	public Disk(){

	}

	public Disk(String type, int capacity, int price){
		this.type = type;
		this.capacity = capacity;
		this.price = price;
	}

	//setter dan getter atribut type
	public void setType(String type){
		this.type = type;
	}

	public String getType(){
		return this.type;
	}

	//setter dan getter atribut capacity
	public void setCapacity(int capacity){
		this.capacity = capacity;
	}

	public int getCapacity(){
		return this.capacity;
	}

	//setter dan getter atribut price
	public void setPrice(int price){
		this.price = price;
	}

	public int getPrice(){
		return this.price;
	}

}

File: Latihan B/PHP/Disk.php
<?php

class Disk{

	//atribut private
	private $type = ""; //tipe disk
	private $capacity = 0; //kapasitas disk
	private $price = 0; //harga disk


	//konstruktor
	public function __construct($type = "", $capacity = 0, $price = 0){
		$this->type = $type;
		$this->capacity = $capacity;
		$this->price = $price;
	}

	//getter dan setter atribut type
	public function setType($type){
		$this->type = $type;
	}

	public function getType(){
		return $this->type;
	}

	//getter dan setter atribut capacity
	public function setCapacity($capacity){
		$this->capacity = $capacity;
	}

	public function getCapacity(){
		return $this->capacity;
	}

	//getter dan setter atribut price
	public function setPrice($price){
		$this->price = $price;
	}

	public function getPrice(){
		return $this->price;
	}

	//destruktor
	public function __destruct(){

	}

}

?>